e2e_tests/utils/index.ts

```diff
@@ -32,3 +32,51 @@ export const formatObjectDateToLongFormat = (objectDate: Record<string, number |
 export const changeDateFromLongFormatToShort = (dateToConvert: string) => {
   return DateTime.fromFormat(dateToConvert, 'dd MMMM yyyy').toFormat('dd/MM/yyyy')
 }
+
+export const formatDateToDNTRLetterFormat = objectDate => {
+  const d = objectDate
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ]
+  const weekDays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+  const weekDay = weekDays[d.getDay()]
+  let hr = d.getHours()
+  let min = d.getMinutes()
+  if (min < 10) {
+    min = `0${min}`
+  }
+  let ampm = 'am'
+  if (hr > 12) {
+    hr -= 12
+    ampm = 'pm'
+  }
+  const date = d.getDate()
+  const month = months[d.getMonth()]
+  const year = d.getFullYear()
+  const nth = function (rawDate) {
+    if (rawDate > 3 && rawDate < 21) return 'th'
+    switch (rawDate % 10) {
+      case 1:
+        return 'st'
+      case 2:
+        return 'nd'
+      case 3:
+        return 'rd'
+      default:
+        return 'th'
+    }
+  }
+  return `${weekDay} ${date}${nth(date)} ${month} ${year} at ${hr}:${min}${ampm}`
+}
```

Comment:
This can be achieved using `luxon` as well, did you try this, will save a few lines of code. Its not mandatory to accept this, it's just a suggestion :)

e.g. changes `Thu Jun 29 2023 at 05:22:47 GMT+0100 (British Summer Time)` to `Thursday 29th June 2023 at 5:22AM`

```suggestion
export const formatDateToDNTRLetterFormat = (objectDate: Date) => {
  const nth = function (rawDate) {
    if (rawDate > 3 && rawDate < 21) return 'th'
    switch (rawDate % 10) {
      case 1:
        return 'st'
      case 2:
        return 'nd'
      case 3:
        return 'rd'
      default:
        return 'th'
    }
  }
  cy.log(`objectDate--> ${objectDate.toDateString()} at ${objectDate.toTimeString()}`)
  return DateTime.fromObject({
    year: objectDate.getFullYear(),
    month: objectDate.getMonth() + 1,
    day: objectDate.getDate(),
    weekday: objectDate.getDay(),
    hour: objectDate.getHours(),
    minute: objectDate.getMinutes(),
    second: objectDate.getSeconds(),
  }).toFormat(`EEEE d'${nth(objectDate.getDate())}' MMMM yyyy 'at' h':'mma`)
}
```